docs/UserGuide.md

```diff
@@ -172,7 +178,7 @@ Lists all the orders a store has.
 
 Format: `listo`
 
-### Locating orders by keyword: `findo`
+### Locating orders by keyword and delivery/payment status: `findo`
```

Comment:
Changing this to `Finding order(s)` for the heading will be fine, I think. The functionality is already explained under this section